supplements/extra-life/adventure-with-muk.xml

```diff
@@ -0,0 +1,182 @@
+			<p class="indent"><b><i>Nimble Escape. </i></b>You can take the Disengage or Hide action as a bonus action on each of your turns.</p>
+			<p class="indent"><b><i>Languages. </i></b>You can speak, read, and write Common and Goblin.</p>
+		</description>
+		<sheet display="false"/>
+		<rules>
+			<stat name="dexterity" value="2" requirements="!ID_WOTC_TCOE_OPTION_CUSTOMIZED_ASI" />
+			<stat name="wisdom" value="1" requirements="!ID_WOTC_TCOE_OPTION_CUSTOMIZED_ASI" />
+			<select type="Ability Score Improvement" name="Custom Ability Score Improvement +2 (Dankwood Goblin)" supports="Custom Ability Score Increase 2" requirements="ID_WOTC_TCOE_OPTION_CUSTOMIZED_ASI" />
+			<select type="Ability Score Improvement" name="Custom Ability Score Improvement +1 (Dankwood Goblin)" supports="Custom Ability Score Increase 1" requirements="ID_WOTC_TCOE_OPTION_CUSTOMIZED_ASI" />
+			<stat name="innate speed" value="30" bonus="base"/>
+			<grant type="Size" id="ID_SIZE_SMALL" />
+			<grant type="Vision" id="ID_VISION_DARKVISION" />
+			<grant type="Language" id="ID_LANGUAGE_COMMON" requirements="!ID_WOTC_TCOE_OPTION_CUSTOMIZED_LANGUAGE" />
+			<grant type="Language" id="ID_LANGUAGE_GOBLIN" requirements="!ID_WOTC_TCOE_OPTION_CUSTOMIZED_LANGUAGE" />
+			<select type="Language" name="Customized Language" supports="Custom Race Language" number="2" requirements="ID_WOTC_TCOE_OPTION_CUSTOMIZED_LANGUAGE" />
+			<grant type="Racial Trait" id="ID_WOTC_AWM_RACIAL_TRAIT_DANKWOOD_GOBLIN_SPEAK_WITH_SMALL_BEASTS" />
+			<grant type="Racial Trait" id="ID_WOTC_AWM_RACIAL_TRAIT_DANKWOOD_GOBLIN_NIMBLE_ESCAPE" />
+		</rules>
+	</element>
+	<element name="Speak with Small Beasts" type="Racial Trait" source="Adventure with Muk" id="ID_WOTC_AWM_RACIAL_TRAIT_DANKWOOD_GOBLIN_SPEAK_WITH_SMALL_BEASTS">
+		<description>
+			<p>Through sounds and gestures, you can communicate simple ideas with Small or smaller beasts. Dankwood goblins love animals and often keep squirrels, badgers, rabbits, moles, woodpeckers, and other creatures as beloved pets.</p>
+		</description>
+		<sheet>
+			<description>Through sounds and gestures, you can communicate simple ideas with Small or smaller beasts.</description>
+		</sheet>
+	</element>
+	<element name="Nimble Escape" type="Racial Trait" source="Adventure with Muk" id="ID_WOTC_AWM_RACIAL_TRAIT_DANKWOOD_GOBLIN_NIMBLE_ESCAPE">
+		<description>
+			<p>You can take the Disengage or Hide action as a bonus action on each of your turns.</p>
+		</description>
+		<sheet>
+			<description>You can take the Disengage or Hide action as a bonus action on each of your turns.</description>
+		</sheet>
+	</element>
+	<element name="Birdsquirrel" type="Companion" source="Adventure with Muk" id="ID_WOTC_AWM_COMPANION_BIRDSQUIRREL">
+		<supports>Familiar, Variant Familiar</supports>
```

Comment:
also i know i did basically said you should do it, but after discussing it with others, we think we shouldn't place these supports on new creatures, unless they specifically mention being Familiars. What we gonna instead try to do in a separate pull is to change how "Familiar Selection" item works, so it would automatically grab any creature that's size Tiny.
```suggestion
```